src/msg/async/dpdk/EventDPDK.h

```diff
@@ -16,25 +16,35 @@
 #ifndef CEPH_EVENTDPDK_H
 #define CEPH_EVENTDPDK_H
 
+#include "common/ceph_time.h"
 #include "msg/async/Event.h"
 #include "msg/async/Stack.h"
 #include "UserspaceEvent.h"
 
 class DPDKDriver : public EventDriver {
-  CephContext *cct;
-
+    CephContext *cct;
+    int epfd;
+    int wakeup_fd;
  public:
-  UserspaceEventManager manager;
+    UserspaceEventManager manager;
 
-  explicit DPDKDriver(CephContext *c): cct(c), manager(c) {}
-  virtual ~DPDKDriver() { }
+    explicit DPDKDriver(CephContext *c): cct(c), manager(c) {}
```

Comment:
please avoid the space-only code churn if we are not touching this area.

Reply:
ok